website/i18n/bg/docusaurus-plugin-content-docs/current/Configuration.md

```diff
@@ -66,7 +66,7 @@ Next to the WebDriver based capabilities you can apply browser and vendor specif
 
 Additionally, a useful utility is the Sauce Labs [Automated Test Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator#/), which helps you create this object by clicking together your desired capabilities.
 
-Type: `Object`<br /> Default: `null`
```

Comment:
Can we revert the changes in this file?

Reply:
u've reverted this. I don't know what changed this. maybe the RUNME extension?